core/ecschema-metadata/src/Metadata/Class.ts

```diff
@@ -135,15 +135,15 @@ export abstract class ECClass extends SchemaItem implements CustomAttributeConta
    * Searches, case-insensitive, for a local ECProperty with the name provided.
    * @param name
    */
-  public async getProperty(name: string, includeInherited: boolean = false): Promise<Property | undefined> {
```

Comment:
We need to update the docuemntation here to explicitly call out the behavior of the flag and the default value.

Additionally we need to update the next version md file to include this in the list of breaking changes

Reply:
Update the code docs and nextversion.md